src/Analyzer.JsonRuleEngine/JsonRuleEngine.cs

```diff
@@ -96,6 +147,7 @@ private static List<RuleDefinition> ParseRuleDefinitions(string rawRuleDefinitio
                 {
                     currentRule = rule.Id;
                     rule.Expression = rule.ExpressionDefinition.ToExpression();
+                    rule.Severity = rule.Severity == default ? 2 : rule.Severity;
```

Comment:
Instead of assigning a default value here, do it in the schema directly (see comment in RuleDefinition.cs).